examples/example7a_SAS/example7a.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+#########################################################
+# Example problem of runing SAS4A/SASSY-1 with WATTS.
+# This is a steady-state, single pin problem with uniform 
+# dimensions and constant properties. The fuel pin has a
+# constant linear heat generation and the boundaries of 
+# the problem are the top and bottom of the fuel pin. 
+# Sodium coolant enters from the bottom. The fuel pin is
+# separated into a bottom unheated section, fuel section,
+# gas plenum, and top unheated section. This problem DOES
+# NOT use the PRIMAR4 module.
+#########################################################
+
+from math import cos, pi
+import os
+import watts
+from astropy.units import Quantity
+
+
+params = watts.Parameters()
+
+# Channel params
+params['sas_version'] = 5.5
+params['tmax'] = 400.0 # maximum problem time in s
+params['flow_per_pin'] = 0.15 # kg/s
+params['total_reactor_power'] = Quantity(182, "MW") # 182000000.0 W
```

Comment:
When we're specifying a quantity explicitly, I don't think we need to put a comment afterward -- it should be clear enough as is.